Changes

```diff
@@ -13,6 +13,12 @@ Revision history for Perl extension Net::SSLeay.
 	  LibreSSL 3.1 - 3.3, parts of which are enabled by default, is not
 	  libssl-compatible. See the "KNOWN BUGS AND CAVEATS" section of
 	  lib/Net/SSLeay.pod for details.
+	- Add constants for, but not limited to,
+          SSL_CTX_set_msg_callback and SSL_set_msg_callback functions:
+          SSL3_RT_* for record content types, SSL3_MT_* for Handshake
+          and ChangeCipherSpec message types, SSL2_VERSION to
+          complement the list of existing SSL and TLS version
+          constants and SSL2_MT_* for SSLv2 Handshake messages.
```

Comment:
nit: indentation (I constantly get tripped up by this - we should probably just use leading spaces instead of a tab if that's allowed by the standard)

Reply:
Switched to tabs. I thought I had it correctly but apparently I had done one more reformat which removed the tabs again.